docs/user/guides/administrators.rst

```diff
@@ -19,3 +19,4 @@ have a look at our :doc:`/tutorial/index`.
    deprecating-content
    pdf-non-ascii-languages
    importing-private-repositories
+   Configure Pull Request Builds <pull-requests>
```

Comment:
Seems odd to use a custom title here? If we want another title, let's just change the title of the article.